awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGenerator.java

```diff
@@ -526,6 +529,12 @@ private static void setHttpStatus(SpanData span, AttributesBuilder builder) {
     }
   }
 
+  private static void setSpanKindForDBUser(SpanData span, AttributesBuilder builder) {
+    if (isKeyPresent(span, DB_USER)) {
+      builder.put(AWS_DB_USER, span.getAttributes().get(DB_USER));
+    }
+  }
+
```

Comment:
Two notes:
1. `setSpanKind` doesn't make sense. Revise to `setRemoteDbUser`
2. Per agreed upon design, we should only set `db.user` if the span is a database span. Please add a check for `isDBSpan` before setting this value.